modules/core/src/test/scala/org/tessellation/trust/SybilAttackSuite.scala

```diff
@@ -37,7 +37,6 @@ object SybilAttackSuite extends SimpleIOSuite with Checkers {
 
   def benchmark(allData: List[TrustNode], filterGood: Int => Boolean): ModelBenchmark = {
 
-    val numNodes = allData.size / 2
```

Comment:
why you change something in the SybilAttackSuite? I believe it's not related to rewards 🤔

Reply:
Yeah this should be fixed